graph/src/data/subgraph/schema.rs

```diff
@@ -591,6 +616,80 @@ impl From<super::MappingEventHandler> for EthereumContractEventHandlerEntity {
     }
 }
 
+#[derive(Debug)]
+struct EthereumContractDataSourceTemplateEntity {
```

Comment:
It would be good to have a comment here how these entities are used and how they fit into the overall picture of dynamic data sources. Same for `EthereumContractDataSourceTemplateSourceEntity`